src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1629,13 +1629,13 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
         }
     }
 
-    private function registerMessengerConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader, array $validationConfig)
+    private function registerMessengerConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader, array $validationConfig, PhpFileLoader $phpLoader)
```

Comment:
`XmlFileLoader` is not used in this function anymore. please just change the typhint from XmlFileLoader to PhpFileLoader  without a new argument.